create.js

```diff
@@ -0,0 +1,7 @@
+// reset input-fields of new card
+
+
+function cardReset() {
+  document.getElementById('#card__create').reset()
+  console.log(Hello form-reset!);
+}
```

Comment:
Have a look at this codepen, at my HTML and JavaScript: https://codepen.io/macfrei/pen/mdwOEBd?editors=0010

I know we haven't discussed this in the course yet, but I will give you the solution now anyways 😉: 

1. The `form` element in HTML has a `submit` event we can use
2. We need to select the `form` 
3. The `form` gets an event listener `addEventListener('sumbit', (event) => {...})`
4. We also get an `event` object which we can use
5. Now the `form` can be reset: `form.reset()`

If you have any questions, please get back to me!

Reply:
Cool, thx! I'll implement this 🤓